### PR TITLE
Syndicate Incursion private radio

### PR DESCRIPTION
**Pull request by Cenrus**
## About The Pull Request

Adds a new independent radio channel for the syndicate incursion to use. The syndicate channel cannot hear the incursion one and vice versa, but the incursionists can snoop into syndicate comms by buying an encryption key.

As a consequence, the incursion radio can't hear all channels any more. You will need an encryption key for this.

## Why It's Good For The Game

I want to add Incursion to dynamic at some point and I don't like how they hog up the syndicate channel, taking it away from traitors. Now incursionists and traitors can buy syndicate keys to chat on the same channel. There are also ways for the station to get a hold of syndicate comms during incursion mode, such as the asteroid/lavaland comms agent base.

Opens up some interesting gameplay opportunities, will you be talking with a fellow excommunciate on the syndicate channel, or are you about to be lured into a trap by the team set out to assassinate you?

## Changelog

:cl:
add: Added incursionist private radio channel.
tweak: Excommunicates can buy the syndicate encryption key in the uplink.
/:cl:

### CODE REVIEW

File: code/__DEFINES/radio.dm [hunk 1]
```diff
@@ -64,6 +68,7 @@
 #define FREQ_MEDICAL 1355  //! Medical comms frequency, soft blue
 #define FREQ_ENGINEERING 1357  //! Engineering comms frequency, orange
 #define FREQ_SECURITY 1359  //! Security comms frequency, red
+#define FREQ_INCURSION 1361  //!  Private channel for the incursion team
```
Comment (PowerfulBacon) [inline code/__DEFINES/radio.dm]: This should be lower on the non accessible radio channels rather than in the middle of all the station channels

**Comment by francinum** [comment]
...How many radio tokens do we have left exactly, jeez.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
They need a span for the channel, and their freq needs to be lower and nearer to syndie rather than in the middle of all the station stuff.
(Also they could just use blood brother implants)

**Comment by PowerfulBacon** [comment]
Closing since somewhat stale, feel free to ping and have this branch reopened. This would be better off using the blood brother radio implant, or should use a non-accessible radio channel frequency.

**Pull request closed without merging**